package.json

```diff
@@ -63,6 +63,7 @@
     "@commitlint/config-conventional": "17.0.3",
     "@devlaunchers/eslint-config-bases": "workspace:^",
     "@playwright/test": "1.24.2",
+    "@types/node": "22.2.0",
```

Comment:
Why adding this dep in the global package.json? It should be added to its respective workspace otherwise errors might appear globally